NGitLab/Models/UserQuery.cs

```diff
@@ -0,0 +1,50 @@
+﻿namespace NGitLab
+{
+    public class UserQuery
+    {
+        /// <summary>
+        /// Specifies how many record per paging (Gitlab supports a maximum of 100 projects and defaults to 20).
```

Comment:
```suggestion
        /// Specifies how many records per page (GitLab supports a maximum of 100 items per page and defaults to 20).
```

Reply:
Updated. I also changed the other queries that had the same summary